homeassistant/components/cover/__init__.py

```diff
@@ -45,6 +45,7 @@
 SUPPORT_OPEN = 1
 SUPPORT_CLOSE = 2
 SUPPORT_SET_POSITION = 4
+SUPPORT_GET_POSITION = 6
```

Comment:
We can't change the entity model.

Reply:
Ok, will remove that. 
In which supported_features should I put get_position? SUPPORT_SET_POSITION or OPEN_CLOSE_FEATURES?

Reply:
It's not needed to add it as a feature.